app/views/expenses_additional/edit.html.erb

```diff
@@ -0,0 +1,78 @@
+<% content_for :header_title, 'Expenses' %>
+
+<div class="form-card">
+    <header class="form-card__header">
+      <div class="step-section-header">
+        <div class="step-section-header__subhead">
+          Tell us more about the other expenses you listed.
+        </div>
+      </div>
+    </header>
+
+    <%= form_for @step,
+      as: :step,
+      builder: MbFormBuilder,
+      url: current_path,
+      method: :put do |f| %>
+
+      <div class="form-card__content">
+        <h4 class="step-section-header__headline">
+          Care expenses
+        </h4>
+
+        <%= f.mb_input_field :monthly_care_expenses,
+          "In total, how much do you pay in care expenses each month?" %>
+
+        <%= f.mb_checkbox_set [
+          { method: :childcare,           label: 'Childcare'           },
+          { method: :elderly_care,        label: 'Elderly Care'        },
+          { method: :disabled_adult_care, label: 'Disabled adult care' },
+          ], label_text: 'What kind of care expenses do you have?' %>
```

Comment:
seems like there's some inconsistency in single/double quotes in this file. Think we're doing double?